front-end/src/pages/Dashboard.js

```diff
@@ -1,16 +1,197 @@
-import React from 'react';
-import Project from "../components/Project";
-import AnalyseProject from "../components/AnalyseProject"
+import React from "react";
+import { Helmet } from 'react-helmet-async';
+import { faker } from '@faker-js/faker';
+// @mui
+import { useTheme } from '@mui/material/styles';
+import {Grid, Container, Typography, Card} from '@mui/material';
+// components
+import Iconify from "../dashboard/components/iconify";
+// sections
+import {
+    AppOrderTimeline,
+    AppGhostMeter,
+    AppGhostGraph,
+    AppTrafficBySite,
+    AppCounter,
+    AppConversionRates,
+} from '../dashboard/app';
 
-const Dashboard = () => {
+import SelectProject from "../components/SelectProject";
+import AnalyseProject from "../components/AnalyseProject";
+import Navbar from "../components/Navbar";
 
 
-  return (
-    <div>
-        <Project/> <br/>
-        <AnalyseProject/>
-    </div>
-  );
-};
+export default function Dashboard() {
+    const theme = useTheme();
 
-export default Dashboard;
+    return (
+        <div>
+            <Helmet>
+                <title> Ghostboard </title>
+            </Helmet>
+
+            <Navbar />
+            <br />
+
+            <Container maxWidth="xl">
+                <SelectProject />
+                <AnalyseProject />
+
+                {/*<Typography variant="h5" sx={{ mb: 5 }}>*/}
+                {/*    Analytics for {projectName}*/}
+                {/*</Typography>*/}
+
+                {/*COUNTERS*/}
+                <Grid container spacing={4}>
+                    <Grid item xs={10} sm={1} md={2.3}>
+                        <AppCounter title="Privacy Concerns" total={714000} icon={'ant-design:android-filled'} />
+                    </Grid>
+
+                    <Grid item xs={10} sm={1} md={2.3}>
+                        <AppCounter title="Unsatisfactory Solutions" total={1352831} color="info" icon={'ant-design:frown-outline'} />
+                    </Grid>
+
+                    <Grid item xs={10} sm={1} md={2.3}>
+                        <AppCounter title="Chatbot Repetitions" total={1723315} color="warning" icon={'ant-design:windows-filled'} />
+                    </Grid>
+
+                    <Grid item xs={10} sm={1} md={2.3}>
+                        <AppCounter title="Lengthy Chat Durations" total={45235} color="error" icon={'ant-design:bug-filled'} />
+                    </Grid>
+
+                    <Grid item xs={10} sm={1} md={2.3}>
+                        <AppCounter title="Live Agent Requests" total={234} color="error" icon={'ant-design:bug-filled'} />
+                    </Grid>
+
+                    {/*TODO: Change the graph / make it so that a specific graph appears when you click on a counter*/}
+                    <Grid item xs={12} md={6} lg={8}>
```

Comment:
To avoid/lessen the complexity of components being loaded in the dashboard, you could create a 'counters' component where you load in all the counters, and then add that 'counters' component to this page.